apps/etterlatte-behandling/src/main/kotlin/behandling/etteroppgjoer/forbehandling/EtteroppgjoerBrevService.kt

```diff
@@ -0,0 +1,156 @@
+package no.nav.etterlatte.behandling.etteroppgjoer.forbehandling
+
+import kotlinx.coroutines.coroutineScope
+import no.nav.etterlatte.behandling.etteroppgjoer.EtteroppgjoerBrevMapper
+import no.nav.etterlatte.behandling.klienter.BrevApiKlient
+import no.nav.etterlatte.brev.BrevKlient
+import no.nav.etterlatte.brev.BrevPayload
+import no.nav.etterlatte.brev.BrevRequest
+import no.nav.etterlatte.brev.Pdf
+import no.nav.etterlatte.brev.behandling.mapAvdoede
+import no.nav.etterlatte.brev.behandling.mapInnsender
+import no.nav.etterlatte.brev.behandling.mapSoeker
+import no.nav.etterlatte.brev.behandling.mapSpraak
+import no.nav.etterlatte.brev.hentVergeForSak
+import no.nav.etterlatte.brev.model.Brev
+import no.nav.etterlatte.brev.model.BrevID
+import no.nav.etterlatte.brev.model.oms.EtteroppgjoerBrevData
+import no.nav.etterlatte.grunnlag.GrunnlagService
+import no.nav.etterlatte.libs.common.feilhaandtering.InternfeilException
+import no.nav.etterlatte.libs.common.retryOgPakkUt
+import no.nav.etterlatte.libs.common.sak.Sak
+import no.nav.etterlatte.libs.ktor.token.BrukerTokenInfo
+import java.util.UUID
+
+class EtteroppgjoerBrevService(
+    private val brevKlient: BrevKlient,
```

Comment:
Burde disse merges? Ser ikke helt behovet for to klienter, men det er kanskje noe du har plukket med fra sånn det var fra før?

Reply:
Ja, men tenker jeg ikke tar det i denne PR'en